ResumableUpload.CS.Sample/ResumeableUploadSample.cs

```diff
@@ -0,0 +1,241 @@
+                                    "user",
+                                    CancellationToken.None,
+                                    new Google.Apis.Util.Store.FileDataStore(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Google.Apis.Auth.YouTube")));
+            if (objOAUth2Credential != null )
+                {
+                if (objOAUth2Credential.Token != null)
+                 {
+                    Console.WriteLine("Token Issued: {0}", objOAUth2Credential.Token.Issued);
+                    }
+                }
+            bOKtoContinue =true;
+            }
+        catch (Exception ex)
+            {
+            Console.WriteLine("Google Authorization: {0}", ex.Message);
+            }
+
+            if (bOKtoContinue)
+                {
+                //
+                // Upload a file
+                //
+                try
+                    {
+                    objYouTubeService = new YouTubeService(new BaseClientService.Initializer
+                        {
+                        HttpClientInitializer = objOAUth2Credential,
+                        GZipEnabled = true,
+                        ApplicationName = Assembly.GetExecutingAssembly().GetName().Name
+                        });
+                    //
+                    // Default Timeout is 1 minute 40 seconds. 
+                    // The following changes it to 2 minutes. 
+                    objYouTubeService.HttpClient.Timeout = TimeSpan.FromMinutes(HTTP_CLIENT_TIMEOUT_MINUTES);
+                    bOKtoContinue = true;
+                    }
+                catch (Exception ex)
+                    {
+                    Console.WriteLine(String.Format("YouTubeService error: {0}", ex.Message));
+                    }
+
+                if (bOKtoContinue)
+                    {
+                    Video objVideo  = new Video();
+                    VideoSnippet vs = new VideoSnippet
+                    {
+                        Title = "Test Title",
+                        Description = "Test Description",
+                        Tags = new string[] { "keyword1", "keyword2", "keyword3" },
+                        CategoryId = "25" // News & Politics
+                    };
+                    objVideo.Snippet = vs;
+                    objVideo.Status = new VideoStatus {Embeddable = true,PrivacyStatus = "private"};
+                    objVideo.RecordingDetails = new VideoRecordingDetails 
+                                        {Location = new GeoPoint 
+                                            {Latitude = 41.327464, Longitude = -72.194555, Altitude = 10},
+                                            LocationDescription = "East Lyme, CT, U.S.A.",
+                                            RecordingDate=DateTime.Now.Date};
+                    string strFilePath = "C:\\Users\\YourUserName\\Videos\\Video\\YourVideo.mp4";
```

Comment:
This can be rewritten as:

`@"C:\Users\YourUserName\Videos\Video\YourVideo.mp4";`

String-literals prefixed with an @-symbol don't need to escape backslashes.